src/scrape_up/github/pull_request.py

```diff
@@ -13,6 +13,15 @@ def __scrape_page(self):
         data = requests.get(f"https://github.com/{self.username}/{self.repository}/pull/{self.pr_number}")
         data = BeautifulSoup(data.text,"html.parser")
         return data
+    def labels(self):
+        labels_found=[]
+        data=self.__scrape_page()
+        label_raw=data.find_all("a",class_="IssueLabel hx_IssueLabel width-fit mb-1 mr-1")
+        for d in label_raw:
+            pass
+            labels_found.append(d.get_text().strip())   
+        return labels_found
+        
```

Comment:
Add this in  `try-excpet` block as well, to handle any unwanted error.

Reply:
Okay. What should I add in except block? Can you please let me know what exceptions might occur.

Reply:
Scrapper might not be able to fetch the data, or any other unwanted issue.

Reply:
Done! Kindly review it